src/main/java/org/craftercms/studio/api/v1/util/StudioConfiguration.java

```diff
@@ -115,9 +110,6 @@
 
     /** Security Service */
     String SECURITY_SESSION_TIMEOUT = "studio.security.sessionTimeout";
-    String SECURITY_URLS_TO_INCLUDE = "security.urlsToInclude";
```

Comment:
Why are we removing this? Right now things seem hardcoded here: https://github.com/craftercms/studio/blob/d534f989c957f8f21bc67f90ced47dcab78b2cc4/src/main/java/org/craftercms/studio/impl/v1/web/security/access/StudioGeneralAccessDecisionVoter.java#L37

We would like this to be configurable, no?